Merge Conflict/Assets/Scripts/TagHandler.cs

```diff
@@ -0,0 +1,30 @@
+/**********************************************************************************************************************
+Name:          TagHandler
+Description:   Contains all tags, which are used to identify objects. With CompareTags() its possible to compare the tag 
+               from the given GameObject with a tag from the PossibleTags "list".
+Author(s):     Markus Haubold
+Date:          2024-03-04
+Version:       V1.0
+TODO:          - ongoing: add new tags
+**********************************************************************************************************************/
+
+using UnityEngine;
+
+//add all tags which are used in unity!
+public static class TagHandler
```

Comment:
refactor class to enum + extension methods 
(-> https://stackoverflow.com/a/5985710)